functions/Get-DbaDbAssembly.ps1

```diff
@@ -59,16 +72,24 @@ function Get-DbaDbAssembly {
             } catch {
                 Stop-Function -Message "Error occurred while establishing connection to $instance" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
             }
-
-            foreach ($database in ($server.Databases | Where-Object IsAccessible)) {
+            $databases = $server.Databases | Where-Object IsAccessible
+            if (Test-Bound 'Database') {
+                $databases = $databases | Where-Object Name -in $Database
+            }
+            foreach ($db in $databases) {
                 try {
-                    foreach ($assembly in $database.assemblies) {
+                    $assemblies = $db.assemblies
+                    if (Test-Bound 'Name') {
+                        $assemblies = $assemblies | Where-Object Name -in  $Name
```

Comment:
this could potentially be if `test-bound name, $db.Assemblies[$name] else $db.assemblies`?

Reply:
Yep, I can change that. Was just copying from other functions.

Reply:
fixed in last commit